ui/components/multichain/notification-detail-button/notification-detail-button.tsx

```diff
@@ -45,8 +43,8 @@ export const NotificationDetailButton = ({
       properties: {
         notification_id: notification.id,
         notification_type: notification.type,
-        ...(notification.type !== TRIGGER_TYPES.FEATURES_ANNOUNCEMENT && {
-          chain_id: notification?.chain_id,
+        ...('chain_id' in notification && {
```

Comment:
This is valid. We now have new notifications that are coming down the pipeline that do not support this chain_id field (e.g. web 3 notifications).

In the future, I want us to add some guard/assertion functions that will allow us to better target the type of notifications.

Reply:
I just realised that these are just for segment events. Then this is okay. We can do guard/assertion functions only when components or other logic require it.